deepmd/pt/model/model/sezm_model.py

```
@@ -1440,29 +1521,36 @@ def core_compute_dens(
     @torch.jit.export
     def forward_lower(
         self,
-        extended_coord: Float[Tensor, "nf nall_x3"] | Float[Tensor, "nf nall 3"],
-        extended_atype: Int[Tensor, "nf nall"],
-        nlist: Int[Tensor, "nf nloc nsel"],
-        mapping: Int[Tensor, "nf nall"] | None = None,
+        coord: Float[Tensor, "nf nscatter_x3"] | Float[Tensor, "nf nscatter 3"],
```

Ruff (0.15.17)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)